docs/resources/servicestage_environment.md

```diff
@@ -0,0 +1,91 @@
+---
+subcategory: "ServiceStage"
+---
+
+# huaweicloud_servicestage_environment
+
+Manages an environment resource within HuaweiCloud ServiceStage.
+
+## Example Usage
+
+### Create an environment based on some cce clusters
+
+```hcl
+variable "env_name" {}
+variable "vpc_id" {}
+variable "cce_cluster_id" {}
+
+resource "huaweicloud_servicestage_environment" "test" {
+  name   = var.env_name
+  vpc_id = var.vpc_id
+
+  basic_resources {
+    cce_ids = [
+      var.cce_cluster_id,
+    ]
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) Specifies the region in which to create the ServiceStage environment.
+  If omitted, the provider-level region will be used. Changing this will create a new resource.
+
+* `name` - (Required, String) Specifies the environment name.
+  The name can contain of 2 to 64 characters, only letters, digits, hyphens (-) and underscores (_) are allowed.
+  The name must start with a letter and end with a letter or a digit.
+
+* `vpc_id` - (Required, String, ForceNew) Specifies the VPC ID to which the environment belongs.
+  Changing this will create a new resource.
+
+* `basic_resources` - (Required, List) Specifies the basic resources.
+  The [object](#servicestage_env_basic_resources) structure is documented below.
+
+* `optional_resources` - (Optional, List) Specifies the optional resources.
+  The [object](#servicestage_env_optional_resources) structure is documented below.
+
+* `description` - (Optional, String) Specifies the environment description.
+  The description can contian a maximum of 128 characters.
+
+<a name="servicestage_env_basic_resources"></a>
+The `basic_resources` block supports:
```

Comment:
I would like to just keep the API block here.
```
"base_resources": [
		{
			"id": "523498f1-36c4-11eb-ae36-0255ac1000c2",
			"type": "cce",
			"name": "cce-test"
		}
	]
```

Reply:
done.